tests/integration/workflows/test_matching.py

```diff
@@ -252,3 +252,91 @@ def test_fuzzy_match_without_math_ml_and_latex(workflow_app):
+                "source": "Elsevier B.V.",
+            }
+        ],
+        "arxiv_eprints": [{"categories": ["hep-lat"], "value": "1205.1659"}],
+    }
+
+    obj = workflow_object_class.create(data=record, id_user=1, data_type="hep")
+    eng = object()
+
+    original_abstract = obj.data["abstracts"][0]["value"]
+    assert fuzzy_match(obj, eng)
+    assert "matches" in obj.extra_data
+    matches = get_value(obj.extra_data, "matches.fuzzy")
+
+    # assert the result with correct abstract and not with the incorrect one
+    assert matches[0]["control_number"] == record_1.json["control_number"]
+    assert obj.data["abstracts"][0]["value"] == original_abstract
+    assert not any(record_2.json["control_number"] in sublist for sublist in matches)
```

Comment:
you can simply assert that the length of matches is one and the first match is record_1 (what you already have in your code ;))

Reply:
Keep it simple. As they say xD